functions/convertkit.php

```diff
@@ -74,10 +74,12 @@ function ckgf_convertkit_api_get_forms( $api_key = null ) {
  * @param string $name Subscriber's first name, or full name
  * @param null $api_key ConvertKit API key
  * @param array $fields values for ConvertKit custom fields
+ * @param int $tag Convertkit Tag ID
  *
  * @return array|mixed|object|WP_Error
  */
-function ckgf_convertkit_api_add_email( $form, $email, $name, $api_key = null, $fields = array() ) {
+function ckgf_convertkit_api_add_email( $form, $email, $name, $api_key = null, $fields = array(), $tag ) {
```

Comment:
`$tag` needs to be an optional parameter, as required parameters cannot follow optional ones in PHP 8.x:
https://php.watch/versions/8.0/deprecate-required-param-after-optional

Reply:
Makes sense! I wasn't using PHP 8 back when I first submitted this. Thanks for the catch!